### DISCUSSION TITLE
rAudio 20230528 Release Notes: "•Add - Playback - Display album booklet if available locally (on click album name)"

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#1329] [Fixed Issues]
I'm appreciating the new feature proposed by @kreativkod but it doesn't work on Android.

**Comment by klassikmann** [comment]
Using Firefox or Chrome booklets are displayed, no action if the rAudio App ist used as GUI.

**Comment by ceejayemm** [comment]
How do you make this work - where do you click on the Album name ? In the Playlist, in the Library entry, on the NAS entry, somewhere else ?.  I am using the Brave browser but don't seem able to get this to work.

Chris

**Reply by klassikmann** [reply]
> How do you make this work - where do you click on the Album name ? In the Playlist, in the Library entry, on the NAS entry, somewhere else ?. I am using the Brave browser but don't seem able to get this to work.
> 
> Chris

I'm clicking in the playback window on the album name, the third text line (1. Artist, 2. Title, 3. Album name). This works on Windows, Linux and Android with Firefox and Chome, but not with the rAudio-App on Android.

**Reply by ceejayemm** [reply]
MX-Linux / Brave, MX-Linux / Firefox and Windows 10 / Edge or Windows 10 / Chrome - if I click on the Album name in any of these combinations takes me to Last.FM and the details of the album not the Digital Booklet.  The album in question is Pink Floyd / The Endless River (Deluxe Edition) and I have the original digital booklet ("Digital Booklet - The Endless River.pdf") in the same folder as the the album tracks.  I have also copied this file to "The Endless River (Deluxe Edition).pdf" (also in the same folder) to fully match the album name if that is necessary.  Neither version shows up in rAudio.

Chris

**Reply by klassikmann** [reply]
The booklet has to be named “booklet.pdf", uppercase or lowercase 'b'.

**Reply by ceejayemm** [reply]
Sorted - thanks

Chris

**Comment by klassikmann** [comment]
rAudio 20230609: Feature still not present in the rAudio app (Android).

**Reply by rern** [reply]
[Why are all android web browsers unable to display PDF documents?](https://android.stackexchange.com/questions/249363/why-are-all-android-web-browsers-unable-to-display-pdf-documents)

**Comment by klassikmann** [comment]
@rern 
I have three choices on Android as rAudio GUI trying to display the booklet:

1. Firefox
Firefox is opening a separate tab presenting the booklet and an additional download button.
2. Chrome
Chrome is opening a separate tab (about:blank) **not** presenting the booklet but silently downloading it.
3. rAudio app
The app does nothing.

**Comment by klassikmann** [comment]
**rAudio 20230615**:
On Android the following message in presented:
![Pic](https://github.com/rern/rAudio/assets/25383513/fef06dde-9e28-4953-98bf-f63a88e4b8ee)
I think this is sufficient to close the thread.
